src/TwinGet.AutomationInterface/Exceptions/CouldNotCreateTwincatDteException.cs

```diff
@@ -0,0 +1,20 @@
+﻿// This file is licensed to you under MIT license.
+
+namespace TwinGet.AutomationInterface.Exceptions
+{
+    [Serializable]
+    public class CouldNotCreateTwincatDteException : Exception
```

Comment:
Should we harmonize Exception naming, at the moment there are at least two versions:
1. With exception at the end
2. Without exception at the end

Reply:
I guess the best practice is to have Exception at the end of the class name

Reply:
Yeah all exceptions should have postfix Exception, I will get to it in #8 . Right now there are way too many specific exceptions for that one module.